### ISSUE TITLE
Most snippets mentioned in manual crashes next

### ISSUE THREAD

**Issue by walseb** [atlas-engineer/next#711]
**Describe the bug**
Most code snippets I have put into the config crashes next. This seems to be due to next not importing modules the snippets depends on, but I can't find anywhere where it's mentions that you have to import something before these snippets. Maybe I have missed something. Thanks for any help!

**Precise recipe to reproduce the issue**
Insert pretty much any snippet from here: https://github.com/atlas-engineer/next/blob/master/documents/MANUAL.org
For this example I chose the snippet to enable vi bindings:
```
(define-configuration buffer
  ((default-modes (append '(vi-normal-mode) %slot-default))))
```

**Information**
- OS name+version: nixos+20.03
- Kernel name+version: Linux+5.6.5
- Graphics card and driver: 
  AMD Ryzen 3 2200U with Radeon Vega Mobile Gfx 
  Driver: Doesn't seem to have a name, it's included in the kernel I think
- Desktop environment / Window manager name+version: exwm+lastest release
- Next version (from =M-x next-version= or =next --version=): 1.5.0
- How you installed Next (Guix pack, package manager, build from source):nix
- Lisp implementation/version (if built from source):
- WebKitGTK+ | QtWebEngine version: I don't know, whatever nix builds against

About the WebKitGTK+ version:  it's _not_ 4.0, current version as of March 2020
is 2.28.

**Output when started from a shell**
```
Next version 1.5.0
Loading configuration from #P"/home/admin/.config/next/init.lisp"...

; file: /home/admin/.config/next/init.lisp
; in: DEFINE-CONFIGURATION BUFFER
;     ((DEFAULT-MODES (APPEND '(VI-NORMAL-MODE) %SLOT-DEFAULT)))
; 
; caught ERROR:
;   illegal function call

; file: /home/admin/.config/next/init.lisp
; in: DEFINE-CONFIGURATION BUFFER
;     (DEFINE-CONFIGURATION BUFFER
;      ((DEFAULT-MODES (APPEND '(VI-NORMAL-MODE) %SLOT-DEFAULT))))
; 
; caught WARNING:
;   undefined variable: COMMON-LISP-USER::BUFFER
; 
; caught STYLE-WARNING:
;   undefined function: COMMON-LISP-USER::DEFINE-CONFIGURATION
; 
; compilation unit finished
;   Undefined function:
;     DEFINE-CONFIGURATION
;   Undefined variable:
;     BUFFER
;   caught 1 ERROR condition
;   caught 1 WARNING condition
;   caught 1 STYLE-WARNING condition
While evaluating the form starting at line 1, column 0
  of #P"/home/admin/.config/next/init.lisp":
Error: we could not load the init file
The variable BUFFER is unbound.
```

**Comment by gravitational-dark-light** [comment]
Isn't 1.5.0 the stable version? If I remember well, define-configuration was made _after_ 1.5.0, and the docs updated accordingly

**Comment by walseb** [comment]
Ah I see, thanks!

**Comment by Ambrevar** [comment]
This is an issue with the way we currently distribute the current
documentation.
In the (near) future, we will embed the documentation so that users
always get to see the documentation that matches the version they are running.

Sorry for the inconvenience!

**Comment by doncatnip** [comment]
It seems to me that 1.5 is the latest release. The README refers to [1] for documentation. All snipplets I tried from [1] failed. There is no solution in this ticket and yet it is closed. How would one proceed ? All I really want to know is how to switch to vim bindings.

[1] https://github.com/atlas-engineer/next/blob/master/documents/MANUAL.org#customization as

**Comment by Ambrevar** [comment]
Indeed, a drawback of 1.5 is that documentation was not embedded.  This
is fixed with the upcoming 2.0.

If you want to consult 1.5 documentation, see here:
https://github.com/atlas-engineer/next/blob/1.5.0/documents/MANUAL.org

If you've got the time to try it, give the 2.0 (master) a shot, it's far
better than 1.5 :)

If you can't figure out how to set up VI bindings, let me know which
version you are using and I'll show you the right snippet.

Cheers!

**Comment by doncatnip** [comment]
Ah cool, thanks ! Such a cool project, glad to see major iterations going on ! :)
